.zappr.yaml

```diff
@@ -0,0 +1,3 @@
+X-Zalando-Team: "pathfinder"
```

Comment:
I think for opensource we need more:

https://github.com/zalando/patroni/blob/master/.zappr.yaml